.readthedocs.yaml

```diff
@@ -12,15 +12,11 @@ build:
     python: "3.11"
   jobs:
     post_create_environment:
-      - pip install poetry==1.8.2
+      - pip install uv
     post_install:
-      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH poetry install --with develop
-      # Debugging CI build
-      - pip freeze
-      - mkdocs --version
-      - python -m site
-      - poetry run mkdocs --version
+      - uv sync --extra develop
```

Comment:
I think that if we have the same `VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH` variable set, `uv sync` should install the requirements to the right location without needing the additional `pip install`s